### PR TITLE
Added "ROM not found." message if specified file is invalid.

### LINKED ISSUE #590: Specifying a game path that doesn't exist prints no error message
Launching Ryujinx with a path to a game directory or image that doesn't exist prints no log messages in the console, and the window remains open as if it's loading. This can be confusing, and should print a "ROM not found" message.

### PR DESCRIPTION
**Pull request by AlexLavoie42**
Fixes #590

### CODE REVIEW

File: Ryujinx/Program.cs [hunk 1]
```diff
@@ -75,6 +75,9 @@ static void Main(string[] args)
                             break;
                     }
                 }
+                else {
```
Comment (jduncanator) [inline Ryujinx/Program.cs]: New line after else and before the opening brace.

File: Ryujinx/Program.cs [hunk 2]
```diff
@@ -75,6 +75,9 @@ static void Main(string[] args)
...
+                    Logger.PrintInfo(LogClass.Application, "ROM not found.");
```
Comment (BaronKiko) [inline Ryujinx/Program.cs]: This message could do with more detail.
Like the other message below. Something like ""Please specify the game XCI/NSP". Make sure to include all supported formats.
Also this should probably be a warning, not info as you can't do much without a game.

Comment (AlexLavoie42) [reply Ryujinx/Program.cs]: Ya good point, will do.

Comment (jduncanator) [reply Ryujinx/Program.cs]: The other message about no arg passed is an Info as well, if we change one we should change both.

Comment (AlexLavoie42) [reply Ryujinx/Program.cs]: > The other message about no arg passed is an Info as well, if we change one we should change both.

Ya that's why I made it just Info. Should I change them both to warnings or errors?

Comment (jduncanator) [reply Ryujinx/Program.cs]: Make them both warnings. Error is too in your face, imo

Comment (AlexLavoie42) [reply Ryujinx/Program.cs]: Ok true.

**Comment by AlexLavoie42** [comment]
Not sure if I got all the supported extensions

**Comment by BaronKiko** [comment]
lgtm :)

**Pull request merged**